src/Features/Core/Portable/CodeFixes/DiagnosticExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeFixes.Suppression;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CodeFixes
+{
+    internal static class DiagnosticExtensions
+    {
+        public static bool GreaterThanOrEqualTo(this DiagnosticSeverity left, DiagnosticSeverity right)
```

Comment:
😕 Any reason to not use `>=` directly on the enum values?
https://sharplab.io/#v2:EYLgtghglgdgNAExAagD4AEBMBGAsAKHQGYACLEgYRIG8CT6zTgB7ZgGxIFkAKAURIhwS/YAEoadBlPQB2ASQB8AXhLAA3JPoBfAjvwFiJAKYwArmGEFa+KQEE4mkgCEH+LUA===

Reply:
I will withdraw my objection if the method is renamed to be clear that the comparison is functionally a semantic comparison (e.g. `IsMoreSevereThanOrEqualTo`).